roles/jenkins_master/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - role: certificate_authority
+  - role: repositories
+  # TODO: consider adding jenkins_common role as a dependency
```

Comment:
It is, we would need to modify the jenkins_master role to use the jenkins_common role.  I was thinking we could defer that work into a separate issue since it may require a decent amount of modification and testing to make happen.